drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -166,13 +177,16 @@ protected void setInstanceHeaderMetaData(ScenarioGridColumn scenarioGridColumn,
      * @param propertyClass it contains the full classname of the property (eg. com.Author)
      * @param propertyHeaderTitle The title to assign to this property. Can be null, in this case it will be retrieved used <code>getPropertyHeaderTitle()</code> method
      */
-    protected void setPropertyHeader(ScenarioSimulationContext context, ScenarioGridColumn selectedColumn, List<String> propertyNameElements, String propertyClass, Optional<String> propertyHeaderTitle) {
+    protected void setPropertyHeader(ScenarioSimulationContext context, ScenarioGridColumn selectedColumn, List<String> propertyNameElements, String propertyClass, String propertyHeaderTitle) {
         int columnIndex = context.getModel().getColumns().indexOf(selectedColumn);
         String aliasName = propertyNameElements.get(0);
         String canonicalClassName = getFullPackage(context) + aliasName;
+        if (selectedColumn.isInstanceAssigned() && !aliasName.equals(selectedColumn.getInformationHeaderMetaData().getTitle())) {
```

Comment:
Can you please add a comment to clarify this? Maybe `aliasName` should be renamed as `instanceAliasName`

Reply:
Done!